twilio-audio-router/build.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion 29
```

Comment:
Can we inherit these from anywhere else that is top-level? If not that's fine for now, just double-checking. I have a vague recollection of doing this in `twilio-video-android`.

Reply:
I thought about that, but I don't think that would make sense for this module as it will eventually get extracted into a standalone repo as an external library.